src/audiomixerboard.cpp

```diff
@@ -1503,11 +1548,16 @@ void CAudioMixerBoard::SetChannelLevels ( const CVector<uint16_t>& vecChannelLev
 {
     const int iNumChannelLevels = vecChannelLevel.Size();
     int       i                 = 0;
+    float     alpha             = AUTO_FADER_ADJUST_ALPHA;
```

Comment:
Why do this?

Reply:
This is just to be able to use the short `alpha` below instead the bulky define.

I will move `alpha` directly to the code line concerning the averaging:
```
            const float alpha = AUTO_FADER_ADJUST_ALPHA;
            vecAvgLevels[iChId] = (1.0f - alpha) * vecAvgLevels[iChId] +
                alpha * vecChannelLevel[i];
```

Or would you prefer removing `alpha` completely and using `AUTO_FADER_ADJUST_ALPHA` instead?